requirements/edx/github.txt

```diff
@@ -40,8 +40,8 @@ git+https://github.com/edx/rfc6266.git@v0.0.5-edx#egg=rfc6266==0.0.5-edx
 -e git+https://github.com/edx/event-tracking.git@0.2.0#egg=event-tracking
 -e git+https://github.com/edx-solutions/django-splash.git@7579d052afcf474ece1239153cffe1c89935bc4f#egg=django-splash
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@release-2015-08-19T17.17#egg=edx-ora2
--e git+https://github.com/edx/edx-submissions.git@7c766502058e04bc9094e6cbe286e949794b80b3#egg=edx-submissions
+-e git+https://github.com/edx/edx-ora2.git@release-2015-08-21T16.14#egg=edx-ora2
+-e git+https://github.com/edx/edx-submissions.git@906c7b7a27e1a8684834334a4bf7fb50a3aede30#egg=edx-submissions
```

Comment:
As part of this work, it would be great to move away from installing these as editable dependencies. I'm not sure when in the upgrade process doing so will introduce the least friction, but it should be kept in mind.